aether-kernel/entrypoint.sh

```diff
@@ -75,7 +75,7 @@ setup () {
     ./manage.py migrate --noinput
 
     # Create readonly database user
-    python /code/sql/create_readonly_user.py
+    python /code/sql/grant_readonly_user.py
```

Comment:
Remove this line and execute the script outside the container. The readonly user is not necessary to start the app.
Do not change the command, leave it as it is now. We can fix it afterwards.

Reply:
What do you mean by "outside the container"? Where exactly? This has to be run after the migrations but before the app starts. Besides, the app is the one that has the knowledge about the tables involved, so I would prefer to have this code within aether and not in k8s.